assets/controller.yaml

```diff
@@ -108,6 +108,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
```

Comment:
I don't actually see `spec.nodeName` set anywhere, am I missing something obvious?

Reply:
These are supported in "fieldRef" by default, like we have metadata.namespace,status.podIP etc.